src/bundler.zig

```diff
@@ -1027,7 +1027,7 @@ pub const Bundler = struct {
 
                     if (bundler.fs.fs.needToCloseFiles()) {
                         file.close();
-                        file_op.fd = 0;
+                        file_op.fd = .zero;
```

Comment:
i havent looked over every usage of .zero, but i suspect most of them can become `invalid_fd`